Makefile

```diff
@@ -0,0 +1,14 @@
+all: fmt build
+
+fmt:
+	go fmt ...
+
+build:
```

Comment:
Removed the stamp thing. I would not care too much about optimizing via Makefile logic. go usually does caching itself. Best to only use Makefiles as entrypoints.

Reply:
> Removed the stamp thing. I would not care too much about optimizing via Makefile logic. go usually does caching itself. Best to only use Makefiles as entrypoints.

The `Makefile` is for integration with libvirt and other projects that'll want to consume the C interface, see `Makefile.*.inc.am` in https://github.com/virtblocks/libvirt/tree/virtblocks/src/golang for an example.

The use of `stamp` was an attempt to automatically trigger recompilation of the Go code when necessary, but I didn't really spend any time on it and in fact was not working properly at all, so I'm okay with ditching it for the time being and coming up with something better later.